Fails to build

## System Information
**OS: Fedora 43 (netinstall)**  
**DE: KDE Plasma 6.6.3**  

## Logs / Crash Report  
```stast@stast-pc ~/З/TeamFortress2-Linux-Internal-main> ./build.sh
mkdir: невозможно создать каталог «libs/funchook»: Нет такого файла или каталога
Клонирование в «libs/funchook»...
remote: Enumerating objects: 1239, done.
remote: Counting objects: 100% (428/428), done.
remote: Compressing objects: 100% (111/111), done.
remote: Total 1239 (delta 350), reused 317 (delta 317), pack-reused 811 (from 2)
Получение объектов: 100% (1239/1239), 348.94 KiB | 976.00 KiB/s, готово.
Определение изменений: 100% (817/817), готово.
-- The C compiler identification is GNU 15.2.1
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test GNU_SPECIFIC_STRERROR_R
-- Performing Test GNU_SPECIFIC_STRERROR_R - Success
-- Performing Test HAVE_FVISIBILITY_HIDDEN
-- Performing Test HAVE_FVISIBILITY_HIDDEN - Success
CMake Warning (dev) at /usr/share/cmake/Modules/FetchContent.cmake:1953 (message):
  Calling FetchContent_Populate(distorm) is deprecated, call
  FetchContent_MakeAvailable(distorm) instead.  Policy CMP0169 can be set to
  OLD to allow FetchContent_Populate(distorm) to be called directly for now,
  but the ability to call it with declared details will be removed completely
  in a future version.
Call Stack (most recent call first):
  CMakeLists.txt:122 (FetchContent_Populate)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done (2.0s)
-- Generating done (0.0s)
-- Build files have been written to: /home/stast/Загрузки/TeamFortress2-Linux-Internal-main/libs/funchook
[  6%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/decoder.c.o
[ 13%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/distorm.c.o
[ 20%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/instructions.c.o
[ 26%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/insts.c.o
[ 33%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/mnemonics.c.o
[ 40%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/operands.c.o
[ 46%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/prefix.c.o
[ 53%] Building C object CMakeFiles/distorm.dir/_deps/distorm-src/src/textdefs.c.o
[ 60%] Linking C static library libdistorm.a
[ 60%] Built target distorm
[ 66%] Building C object CMakeFiles/funchook-static.dir/src/funchook.c.o
[ 73%] Building C object CMakeFiles/funchook-static.dir/src/arch_x86.c.o
[ 80%] Building C object CMakeFiles/funchook-static.dir/src/os_unix.c.o
[ 86%] Building C object CMakeFiles/funchook-static.dir/src/disasm_distorm.c.o
[ 93%] Building ASM object CMakeFiles/funchook-static.dir/src/prehook-x86_64-sysv.S.o
[100%] Linking C static library libfunchook.a
[100%] Built target funchook-static
rm -f obj/main.cpp.o obj/libsigscan/libsigscan.c.o obj/imgui/imgui_tables.cpp.o obj/imgui/imgui_draw.cpp.o obj/imgui/imgui_impl_sdl2.cpp.o obj/imgui/imgui_demo.cpp.o obj/imgui/imgui_impl_opengl3.cpp.o obj/imgui/imgui_impl_vulkan.cpp.o obj/imgui/imgui_widgets.cpp.o obj/imgui/imgui.cpp.o obj/imgui/imgui_stdlib.cpp.o obj/hacks/navbot/micropather/micropather.cpp.o 
rm -f tf2.so
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/main.cpp.o main.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/libsigscan/libsigscan.c.o libsigscan/libsigscan.c
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_tables.cpp.o imgui/imgui_tables.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_draw.cpp.o imgui/imgui_draw.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_impl_sdl2.cpp.o imgui/imgui_impl_sdl2.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_demo.cpp.o imgui/imgui_demo.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_impl_opengl3.cpp.o imgui/imgui_impl_opengl3.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_impl_vulkan.cpp.o imgui/imgui_impl_vulkan.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_widgets.cpp.o imgui/imgui_widgets.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui.cpp.o imgui/imgui.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/imgui/imgui_stdlib.cpp.o imgui/imgui_stdlib.cpp
g++ -shared -fPIC -std=c++23 -g --no-gnu-unique -c -o obj/hacks/navbot/micropather/micropather.cpp.o hacks/navbot/micropather/micropather.cpp
In file included from hooks/../gui/menu.hpp:15,
                 from hooks/sdl.cpp:12,
                 from main.cpp:39:
hooks/../gui/../hacks/pishock/pishock.hpp:5:10: фатальная ошибка: curl/curl.h: Нет такого файла или каталога
    5 | #include <curl/curl.h>
      |          ^~~~~~~~~~~~~
компиляция прервана.
make: *** [Makefile:35: obj/main.cpp.o] Ошибка 1
make: *** Ожидание завершения заданий…
stast@stast-pc ~/З/TeamFortress2-Linux-Internal-main>
```

## Description  
Fails to build, couldnt find curl.h

## Steps to reproduce  
- [X] I am able to consistently reproduce this error

Steps to reproduce the behavior:  
1. Install all packages through script (as intended)
2. Try to build internal in (Fedora Minimal)[https://fedoraproject.org/misc/#minimal], maybe that matters
3. Fails to build

## Checklist  
- [X] I have included my OS and DE or WM.
- [X] I have included the steps to reproduce and my logs to the best I am able to share.
- [ ] The Developers are allowed to contact me on Discord/Telegram/Revolt/etc. for further information if they need it. (im staying anonymous, sry)

--- a/packages/distros/fedora.sh
+++ b/packages/distros/fedora.sh
@@ -1,7 +1,10 @@
 #!/bin/bash
-sudo dnf -y install git cmake gdb gcc g++ make SDL2-devel glew-devel vulkan-loader-devel execstack
-wget https://ftp.lysator.liu.se/pub/opensuse/distribution/leap/15.6/repo/oss/x86_64/libGLEW2_1-2.1.0-1.28.x86_64.rpm #fedora doesnt have libglew 2.1
-sudo rpm -ivh libGLEW2_1-2.1.0-1.28.x86_64.rpm
-rm libGLEW2_1-2.1.0-1.28.x86_64.rpm
+sudo dnf -y install git cmake gdb gcc g++ make SDL2-devel vulkan-loader-devel execstack libcurl-devel
+wget https://cdn.amazonlinux.com/al2023/core/guids/9cf1057036ef7d615de550a658447fad88617805da0cfc9b854ba0fb8a668466/x86_64/../../../../blobstore/218da5cb73cce7b5c684fef7eb03989f0c98601586fa105c5b254e59c8ea8e73/libGLEW-2.1.0-9.amzn2023.0.2.x86_64.rpm # fedora doesnt have libglew 2.1
+wget https://cdn.amazonlinux.com/al2023/core/guids/9cf1057036ef7d615de550a658447fad88617805da0cfc9b854ba0fb8a668466/x86_64/../../../../blobstore/1f27b39ccf0300e88e092f06f28cb98e839929b4a842e98fdf7f4822f15d27b3/glew-devel-2.1.0-9.amzn2023.0.2.x86_64.rpm # fedora doesnt have glew-devel 2.1 anymore
+sudo rpm -ivh libGLEW-2.1.0-9.amzn2023.0.2.x86_64.rpm
+sudo rpm -ivh glew-devel-2.1.0-9.amzn2023.0.2.x86_64.rpm
+rm libGLEW-2.1.0-9.amzn2023.0.2.x86_64.rpm
+rm glew-devel-2.1.0-9.amzn2023.0.2.x86_64.rpm
 sudo dnf -y group install "c-development"
 sudo dnf -y group install "development-tools"

